### DISCUSSION TITLE
Run different versions of laravel project on a single machine

### DISCUSSION THREAD

**Discussion by abdullahrather** [community/community#82998] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

is there a way to run two different versions of laravel project at the same time in windows

**Comment by AbdullahAssi** [comment]
Check this article sir. It may help you out

https://medium.com/@shubhamsunny/how-to-run-multiple-laravel-projects-at-the-same-time-problem-solution-94ccd8cf19cf#:~:text=Open%20your%20command%20prompt%20and,where%20your%20Laravel%20project%20exists.&text=Now%20type%20http%3A%2F%2Flocalhost,your%20blog%20project%20will%20run.&text=Now%20go%20to%20your%20browser,another%20project%20will%20also%20run.

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @abdullahrather, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by supuna97** [comment, marked as answer]
Can be run using **windows** and **wsl**. Same machine but different platforms. 

Run one project in Windows according to the **PHP version**. In the other project, install the **PHP version** related to that project in **wsl** and run it through **wsl**. Then you can run 2 laravel projects of two different versions on the same machine.

**Reply by abdullahrather** [reply]
Thanks Brother much appreciated!
